### ISSUE TITLE
improving navbar UI

### ISSUE THREAD

**Issue by Illaf** [akshitagupta15june/PetMe#736]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Illaf** [comment]
The navbar looks a bit crowded .The contact us button is a bit bigger than usual. 'Powered by translate' is hardly visible. The right corner is looking congested. Please assign this issue so that I can improve the above mentioned things and a lot more that is required. Hope you will not be disappointed.

**Comment by akanksha984** [comment]
I would like to work upon this issue. Please assign it to me Gssoc'23

**Comment by meghanareddy1808** [comment]
Hello @akshitagupta15june,@swethatheresa @AnishaSingh0118,
I want to contribute to this project under GSSoc 2023, please assign me this issue.

**Comment by AnishaSingh0118** [comment]
> I would like to work upon this issue. Please assign it to me Gssoc'23

Are you still working on this issue ?

**Comment by AnishaSingh0118** [comment]
> Hello @akshitagupta15june,@swethatheresa @AnishaSingh0118, I want to contribute to this project under GSSoc 2023, please assign me this issue.

Yes, you can take up this issue.We are open to considering solutions from anyone to solve this issue.
